src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -306,7 +306,7 @@ public async Task GetAsync_StreamRefused_RequestIsRetried()
                 (int streamId1, HttpRequestData requestData1) = await connection.ReadAndParseRequestHeaderAsync(readBody: true);
 
                 // Send SETTINGS frame to change stream limit to 0, allowing us to send a REFUSED_STREAM error due to the new limit
-                connection.ExpectSettingsAck();
+                await connection.ExpectSettingsAckAsync(TimeSpan.FromSeconds(30.0));
```

Comment:
Should all of these 30 second TimeSpans be one static readonly somewhere expressing the recommended default for this?

Reply:
Also, 30 seconds is a really long time in test-land.

Reply:
That's merely reflecting how timeouts are passed in general in that particular test suite, e.g. https://github.com/dotnet/corefx/blob/c663f3ff12bcd95c54e9106aee218671d06fd675/src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs#L85